zulipterminal/ui_tools/boxes.py

```diff
@@ -680,10 +702,8 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                     self.msg_edit_state = None
                     self.keypress(size, primary_key_for_command("GO_BACK"))
         elif is_command_key("GO_BACK", key):
-            self.msg_edit_state = None
-            self.msg_body_edit_enabled = True
-            self.compose_box_status = "closed"
             self.send_stop_typing_status()
+            self._set_compose_attributes_to_defaults()
             self.view.controller.exit_editor_mode()
             self.main_view(False)
```

Comment:
Would it make sense to remove `main_view` fully now too? That seems part of the reset?

Reply:
Responded on the stream, [#zulip-terminal > Resetting WriteBox attributes to defaults #T1057](https://chat.zulip.org/#narrow/stream/206-zulip-terminal/topic/Resetting.20WriteBox.20attributes.20to.20defaults.20.23T1057).